src/types/responses.ts

```diff
@@ -135,3 +135,23 @@ export interface PlayTileJSON {
   tile: string;
   connectionId: string;
 }
+
+/**
+ * Interface for INTERACTION_SUCCESS payload received
+ */
+export interface InteractionSuccessJSON {
+  connectionId?: string;
+  playedTiles?: string[];
+  meldType?: string;
+  skipInteraction: boolean;
+}
```

Comment:
there is also a success attribute in this payload if you ever need it:
```ts
export interface InteractionSuccessJSON {
  connectionId?: string;
  playedTiles?: string[];
  meldType?: string;
  skipInteraction: boolean;
  success: true;
}
```

Reply:
Is there error as well?

Reply:
nope, it would be in `PlayedTileInteractionJSON` is the lambda function failed